### PR TITLE
Extended skbuild.md page

### PR DESCRIPTION
**Pull request by oleksandr-pavlyk**
Extended skbuild.md building on sample-data-parallel-extensions repo.

This page builds on sample data-parallel extensions repo: https://github.com/IntelPython/sample-data-parallel-extensions

### CODE REVIEW

File: content/en/details/skbuild.md [hunk 1]
```diff
@@ -15,13 +15,129 @@ DPC++ is an LLVM-based compiler, but the host portion of the binary it produces
...
+# Notice that first evaluation results in JIT-compiling the kernel
+# Subsequent evaluation reuse cached binary
+f0 = kde.cython_kde_eval(cpu_q, x[:, np.newaxis], data[:, np.newaxis], 3e-6)
+
+f1 = kde.cython_kde_eval(gpu_q, x[:, np.newaxis], data[:, np.newaxis], 3e-6)
+
+assert np.allclose(f0, f1)
 ```
 
+Of course a naive NumPy reference implementation can not handle the large size input like used above, running out of memory,
```
Comment (diptorupd) [inline content/en/details/skbuild.md]: reads a bit awkward. do you mean Numpy will run out of memory?

**Comment by diptorupd** [comment]
maybe one last paragraph on some perf. numbers or some other conclusion is needed. But, do not sweat.

File: content/en/details/skbuild.md (hunk 1 above)
Comment (oleksandr-pavlyk) [reply content/en/details/skbuild.md]: Yes with 

```
MemoryError: Unable to allocate 104. GiB for an array with shape (14000, 1000000, 1) and data type float64
```

Comment (oleksandr-pavlyk) [reply content/en/details/skbuild.md]: Tried to refine.

**Pull request merged**